core/controllers/tasks.py

```diff
@@ -139,6 +140,25 @@ class InstantFeedbackMessageEmailHandler(
 ):
     """Handles task of sending feedback message emails instantly."""
 
+    URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'user_id': {
```

Comment:
I think regex can be added for checking the pattern for the user ID. Can you please take a look into it?

Reply:
Thank you. Found a validator for this in `feconf`. Also increased test coverage testing the validation method.